### PR TITLE
2/how to run k8ssandra locally on k3d

### PR DESCRIPTION
**Pull request by patchandpray**
This fixes issue number 2 which concerns having documentation around setting up a local development environment that can run multiple kubernetes clusters and run k8ssandra.

This guide helps setup k3d, kubectl and deploys only the k8ssandra operator and a simple datacenter with local-path storage. This can be the entrypoint for further development and building familiarity with k3d and k8ssandra.

### CODE REVIEW

**Review by helper-uttam — APPROVED** [review]
LGTM 👍  Great work :)

**Review by Polokghosh53 — APPROVED** [review]

**Review by Polokghosh53 — COMMENTED** [review]
LGTM too. 👍🤝

**Review by shreemaan-abhishek — COMMENTED** [review]
LGTM, any other work left to do before merging this?

File: resources/run-k8ssandra-on-k3d-locally-lightweight/example-cassdc-minimal-local-path-storage.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      accessModes:
+        - ReadWriteOnce
+      resources:
+        requests:
+          storage: 1Gi
+  config:    
+    cassandra-yaml:
+      authenticator: org.apache.cassandra.auth.PasswordAuthenticator
+      authorizer: org.apache.cassandra.auth.CassandraAuthorizer
+      role_manager: org.apache.cassandra.auth.CassandraRoleManager
```
Comment (AbhijithGanesh) [inline resources/run-k8ssandra-on-k3d-locally-lightweight/example-cassdc-minimal-local-path-storage.yaml:26]: Can you mark the reference documents for these YML files, can they be added to our documentation

File: resources/run-k8ssandra-on-k3d-locally-lightweight/local-path-config-relaxed-permissions.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+data:
+  config.json: |-
+    {
+      "nodePathMap":[
+      {
+        "node":"DEFAULT_PATH_FOR_NON_LISTED_NODES",
+        "paths":["/var/lib/rancher/k3s/storage"]
+      }
+      ]
+    }
```
Comment (AbhijithGanesh) [inline resources/run-k8ssandra-on-k3d-locally-lightweight/local-path-config-relaxed-permissions.yaml:11]: It'd be of great use if you can mention the documentation of these references

**Review by AbhijithGanesh — APPROVED** [review]

**Pull request merged**